src/routes/AccountDetailsContainer.js

```diff
@@ -87,13 +94,49 @@ class AccountDetailsContainer extends React.Component {
     return groups.filter(g => g.id === user.patronGroup)[0] || {};
   }
 
+  getItemDetails = () => {
+    const account = this.getAccount();
+    const { resources } = this.props;
+    const loanRecords = resources?.loans?.records ?? [];
+    const itemId = account?.itemId;
+    const item = loanRecords.filter((loan) => loan.itemId === itemId);
+    const contributorsRecord = item[0]?.item?.contributors ?? [];
```

Comment:
Sorry, I missed it yesterday.  It seems, contributorsRecord can contain information about many contributors. Please, rename it.